Documentation/concepts/kubernetes/compatibility-table.rst

```diff
@@ -116,7 +118,11 @@
 +-----------------+----------------+
 | v1.9.1          | 1.22.3         |
 +-----------------+----------------+
+| v1.9.2          | 1.22.3         |
++-----------------+----------------+
+| v1.9.3          | 1.22.3         |
++-----------------+----------------+
 | v1.9            | 1.22.3         |
 +-----------------+----------------+
-| latest / master | 1.22.3         |
+| latest / master | 1.22.4         |
```

Comment:
Drawing attention to this. Not sure if this was done correctly, would appreciate a look over @aanm

Reply:
Looks right to me at a glance of `pkg/k8s/apis/cilium.io/v2/register.go`, though I do kinda wonder why we even have a `latest / master` row in this table since that's subject to change arbitrarily at any point in time as we develop additional features.